optimum/onnxruntime/trainer.py

```diff
@@ -818,7 +821,7 @@ def get_dataloader_sampler(dataloader):
                 self.control.should_training_stop = True
 
             self.control = self.callback_handler.on_epoch_end(args, self.state, self.control)
-            self._maybe_log_save_evaluate(tr_loss, grad_norm, model, trial, epoch, ignore_keys_for_eval)
+            self._maybe_log_save_evaluate(tr_loss, grad_norm, model, trial, epoch, ignore_keys_for_eval, start_time)
```

Comment:
`start_time` was introduced in transformers v4.47 https://github.com/huggingface/transformers/blob/v4.47.0/src/transformers/trainer.py#L3021 so maybe we should add a check for the transformers version (min / max supported version) + tests to make sure we keep compatibility with the min/max versions, wdyt @IlyasMoutawwakil ?